cfme/services/requests.py

```diff
@@ -197,6 +197,30 @@ def debug_requests():
             logger.debug(' {}'.format(row))
 
 
+def go_to_request_page(cells):
+    """Finds the request and opens the page
```

Comment:
I understand, but the test below is called go_to_request, which _does_ click. Your one is called go_to_request_page and doesn't click, I think may ```find_request``` may be better.